sim/rogue/item_sets_phase_5.go

```diff
@@ -165,11 +165,11 @@ func (rogue *Rogue) applyT2Tank4PBonus() {
 	rogue.RegisterAura(core.Aura{
 		Label: label,
 		OnInit: func(aura *core.Aura, sim *core.Simulation) {
-			oldOnStacksChange := rogue.RollingWithThePunchesProcAura.OnStacksChange
-			rogue.RollingWithThePunchesProcAura.OnStacksChange = func(aura *core.Aura, sim *core.Simulation, oldStacks int32, newStacks int32) {
-				oldOnStacksChange(aura, sim, oldStacks, newStacks)
-				rogue.AddStatDynamic(sim, stats.Armor, float64(0.2*initarmor*float64(newStacks-oldStacks)))
-			}
+			rogue.RollingWithThePunchesProcAura.ApplyOnStacksChange(func(aura *core.Aura, sim *core.Simulation, oldStacks int32, newStacks int32) {
+				if newStacks <= 5 && oldStacks <= 5 {
+					rogue.AddStatDynamic(sim, stats.Armor, float64(0.2*initarmor*float64(newStacks-oldStacks)))
+				}
+			})
```

Comment:
Hm I'm not sure this quite works? If you're at 100% additional armor and then go from 10 stacks => 0 it should remove 5 stacks worth of armor, but because of the condition it won't